tests/Doctrine/Tests/DBAL/Query/QueryBuilderTest.php

```diff
@@ -863,4 +863,23 @@ public function testGetParameterTypes()
 
         $this->assertSame(array('name' => \PDO::PARAM_STR, 'isActive' => \PDO::PARAM_BOOL), $qb->getParameterTypes());
     }
+
+    /**
+     * @group DBAL-790
+     */
+    public function testSelectWithJoinsWithNotUniqueAliases()
```

Comment:
is the issue about a not unique alias or about a circular reference between a and b ?